### ISSUE TITLE
Soundness bug in QF_UF

### ISSUE THREAD

**Issue by numairmansur** [SRI-CSL/yices2#167]
Hi, 
On the following satisfiable formula, yices return **unsat**
[38.smt2](https://github.com/SRI-CSL/yices2/files/4195076/38.smt2.zip)

```
$ ./yices_smt2 38.smt2
unsat
```


z3 and CVC4 return sat


Yices 2.6.2                                                                                                                                                                                                                                                                    
Copyright SRI International.                                                                                                                                                                                                                                                   
Linked with GMP 6.1.2                                                                                                                                                                                                                                                          
Copyright Free Software Foundation, Inc.                                                                                                                                                                                                                                       
Build date: 2020-02-12                                                                                                                                                                                                                                                         
Platform: x86_64-pc-linux-gnu (release)                                                                                                                                                                                                                                        
Revision: 1526d93178a9b39a85d9933eed14e3af76402cb7

**Comment by dddejan** [comment]
Thanks for the report.

In debug mode, we get assertion failure.

```console
./yices_smt2 38.smt2 
Assertion failed: (n > 0 && sorted_array(c, n)), function constant_is_in_set, file context/symmetry_breaking.c, line 1170.
```

**Comment by BrunoDutertre** [comment]
This was a bug in the symmetry breaking code. It's fixed in b5f1d0a7.
